db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Answer.delete_all
+ApplicationRecord.delete_all
+Keyword.delete_all
+Officer.delete_all
+Question.delete_all
+Rso.delete_all
+
+rso.create!(name: 'Association for Computer Machinery',
+            shortName: 'ACM',
+            category: 'Computer Science')
+
+rso.create!(name: 'Knitting Club',
```

Comment:
And I'll need to create Answer_keyword, Officer_rso, and Keyword_rso tables as well?